### PR TITLE
Confirmation button

### PR DESCRIPTION
**Pull request by mahcloud**
added @brettv 's confirmation button to obs-utils

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +14,7 @@ On an existing application, just update the version tag in the `package.json` fi
 
 ## Running Test App
 
-* `brunch watch --server` - runs on http://localhost:3333/
+* `brunch watch --server` - runs on http://localhost:3334/
```
Comment (mahcloud) [inline README.md]: I changed the port number for out test_app so it doesn't collide with ats.

File: __tests__/buttons/confirm-button.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    const node = ReactDOM.findDOMNode(comp);
+
+    TestUtils.Simulate.click(node);
+    expect(comp.state.confirm).toEqual(true);
+    expect(setTimeout.mock.calls.length).toBe(0);
+
+    // Should still be true because mouseOut hasn't fired
+    jest.runAllTimers();
+    expect(comp.state.confirm).toEqual(true);
+    expect(setTimeout.mock.calls.length).toBe(0);
```
Comment (mahcloud) [inline __tests__/buttons/confirm-button.js]: jest time simulator is pretty cool.
[docs](https://facebook.github.io/jest/docs/timer-mocks.html)
allows us to test timeout length based on interactions and execute timeouts without waiting for the time to happen.

File: app/buttons/confirm-button.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    wait: React.PropTypes.number, // time in ms to wait for confirmation
+    onAction: React.PropTypes.func, // function called after confirmation clicked
+    onConfirm: React.PropTypes.func // function called on first click
+  },
+
+  getDefaultProps() {
+    return {
+      label: "Delete",
+      message: "Are you sure?",
+      wait: 3000
```
Comment (mahcloud) [inline app/buttons/confirm-button.js]: Default text and wait time for the confirmation button

File: brunch-config.js [hunk 4]
```diff
@@ -9,6 +9,10 @@ module.exports = {
     }
   },
 
+  server: {
+    port: 3334
```
Comment (mahcloud) [inline brunch-config.js]: This is how to change the port number for brunch server

**Comment by mahcloud** [comment]
@brettv @josephtazworks let me know if you have any feedback

**Comment by brettv** [comment]
That timeout stuff is really cool :-)

**Comment by brettv** [comment]
If you want. It should be pretty trivial to switch between this.props.children and this.props.label

**Pull request merged**